docs/console/design-your-projects/custom-resources/use-cases.md

```diff
@@ -0,0 +1,91 @@
+---
+id: custom-resources-use-cases
+title: Use Cases
+sidebar_label: Use Cases
+---
+
+The following examples showcase how users can leverage **Custom Resources** in different scenarios.
+
+:::tip
+Find out all there is to know about Custom Resources in the [dedicated section](/console/design-your-projects/custom-resources/custom-resources.md).
+:::
+
+## Template-based Custom Resources
+
+### AWS Lambda
```

Comment:
for all these, if we have examples let's put them here

Reply:
added lambda example